src/environments.js

```diff
@@ -216,13 +216,20 @@ defineEnvironment("aligned", {
     let res = {
         type: "array",
         cols: [],
+        rowsAreLines: true,
     };
-    res = parseArray(context.parser, res);
+    res = parseArray(context.parser, res, "display");
```

Comment:
Yes, I checked in LaTeX, and `$\begin{aligned} \frac{x}{y} \end{aligned}$` renders a display fraction.